modules/shared/programs/codex/files/sync-codex-config.py

```
@@ -262,12 +279,17 @@ def _quarantine(reason: str):
     except UnicodeDecodeError as e:
         return _quarantine(f"not valid UTF-8 ({e})")
     try:
-        return tomlkit.parse(text)
+        return tomlkit.parse(text), text
     except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)